src/BenchmarkDotNet/Loggers/LinqPadLogger.cs

```diff
@@ -0,0 +1,72 @@
+using System;
+using System.Collections.Generic;
+using System.Reflection;
+
+namespace BenchmarkDotNet.Loggers
+{
+    public sealed class LinqPadLogger : ILogger
+    {
+        private const string DefaultColor = "";

```

Comment:
It would be better to use an existing color which looks readable in both themes.
It can protect us from a situation when we add new LogKind and forget to update LINQPad color schemes.

Reply:
This works as expected by default; LINQPad sets the default colour to black (in light mode) and white (in dark mode), so everything is readable.